src/views/Login.vue

```diff
@@ -0,0 +1,130 @@
+<template>
+  <v-card class="card">
+    <img class="logo" :src="require('@/assets/images/login-title.png')"/>
+    <div v-if="$apollo.loading" class="loading-wrapper">
+      <v-progress-circular
```

Comment:
When I ran it, the spinner was invisible, even when I crashed the DB-Service so the query should take time I hadn't seen the spinner.